presidio-anonymizer/presidio_anonymizer/anonymizers/fpe.py

```diff
@@ -14,10 +14,14 @@ class FPE(Anonymizer):
     PII text will be replaced with a format preserving encryption using FF1 algorithm.
     """
 
-    def anonymize(self, original_text=None, params={}):
+    def anonymize(self, original_text: str = None, params: dict = None) -> str:
         """Return anonymized text using FF1 algorithm."""
         old_text = params.get("old_text")
         decrypt = params.get("decrypt")
         tweak = params.get("tweak")
         key = params.get("key")
         return old_text + decrypt + tweak + key
```

Comment:
If this is just a placeholder we should add a TODO here.

Reply:
I changed just the method signature

Reply:
Just changed the signature, I guess we are going to implement this anyway in this sprint